test/python/test_resources_api.py

```diff
@@ -254,12 +237,12 @@ def test_get_resource_404a(self):
 
         self.assertEqual(context.exception.status, 404)
 
-    def test_get_resource_404b(self):
+    def test_get_resource_404(self):
```

Comment:
Ah I see. This actually isn't an error in Conjur. I believe what is actually happening is that if a user requests a resource they never had privileges for Conjur will return a 404 for security reasons (we don't want an attacker being able to determine which resources exist). However if you first grant then deny the user privilege to the resource Conjur will return a 403. Or if they have some privilege but not the correct ones then it will also return 403. I dealt with this in these secrets endpoint by first granting then denying privilege. I will change this back to 404b, however we should open a new issue to fix it so we get the 403 code.